yt_dlp/extractor/ard.py

```diff
@@ -577,3 +578,117 @@ def fetch_page(page_num):
         return self.playlist_result(
             OnDemandPagedList(fetch_page, self._PAGE_SIZE), full_id, display_id=display_id,
             title=page_data.get('title'), description=page_data.get('synopsis'))
+
+
+def graphql_query(self, endpoint, display_id, query):
+    return self._download_json(
+        endpoint,
+        display_id,
+        data=json.dumps({'query': '{' + query + '}'}).encode(),
+        headers={
+            'Content-Type': 'application/json',
+        },
+    )['data']
+
+class ARDAudiothekIE(ARDBetaMediathekIE):
```

Comment:
Why subclass from `ARDBetaMediathekIE`? Aside from `_GEO_COUNTRIES` it's not inheriting anything useful from it AFAICT. Don't subclass from concrete IEs anyways; you can set `_GEO_COUNTRIES` in this class too